### ISSUE TITLE
Option to rename the players

### ISSUE THREAD

**Issue by bitboulder** [rnauber/xskat-android#38]
Hi, thank you for this great and simple game app.

It would be nice to have an option to change players names.

Frank

**Comment by jmizv** [comment]
Hi, would this significantly improve your game experience?

**Comment by bitboulder2** [comment]
Yes it would improve my experience because for me Androido and Androida are not easy  distinguishable. This makes some messages confusable.
